### ISSUE TITLE
Block info returned by Indexer does not contain `partupdrmv`

### ISSUE THREAD

**Issue by oliviera9** [algorand/js-algorand-sdk#587]
### Block info returned by Indexer does not contain `partupdrmv`

Take this block, https://algoexplorer.io/block/21595838.

Using the same endpoint for the indexer, the Python SDK `IndexerClient::block_info()` function returns a field
```
'partupdrmv': ['SPACEXXV3RZURE3M3JZY2BESWGUSTCNG2BGAIJXQJHVA5YI5DDVQLF2MSQ']
```
while the Javascript SDK `indexer::lookupBlock()` response does not return nothing similar to `partupdrmv`.
This leads to an erroneous hash calculation for the block.

### Steps to reproduce

1. indexer::lookupBlock(21595838) in Node
2. IndexerClient::block_info(21595838) in Python

### Expected behaviour
The Javascript SDK should return a field containing those addresses indicated in `partupdrmv`.

### Actual behaviour
There is no field resembling `partupdrmv`.

**Comment by jasonpaulos** [comment]
Hi @oliviera9, both the Javascript and Python SDKs directly return the REST response from indexer. I would expect both of them to behave exactly the same with respect to the `partupdrmv` field, meaning they should either both have it or both omit it.

It looks like the underlying problem is that indexer does not expose this field at all. I've made this issue to track it: https://github.com/algorand/indexer/issues/1045

The good news is that algod does return this field, so both the Javascript and Python SDKs should be able to access it from there.

**Comment by winder** [comment]
This would be addressed as part of #1045
